packages/chat-component/src/components/chat-avatar.ts

```diff
@@ -0,0 +1,33 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { styles } from '../styles/chat-avatar.js';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+export interface ChatStage {
+  label: string;
+  svgIcon: string;
+  url: string;
+}
+
+@customElement('chat-avatar')
+export class ChatAvatarComponent extends LitElement {
```

Comment:
since there is nothing in this component that's chat specific, it's a simple linked icon, should we just call it something like that? 'link-icon'. It will make it reusable for any place an clickable icon can be used.

Reply:
Good suggestion. Renamed to 'link-icon'